### PR TITLE
PMM-1710 Add support for Supervisord

### PR DESCRIPTION
**Pull request by AlekSi**
Do not merge yet.

@delgod Makes sense? Should something be added/changed?

### CODE REVIEW

File: service_supervisord_linux.go [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+{{if .UserName}}user {{.UserName}}{{end}}
+
+{{if .WorkingDirectory}}directory {{.WorkingDirectory}}{{end}}
+
+autorestart = true
+
+stdout_logfile = /var/log/{{.Name}}.log
+stderr_logfile = /var/log/{{.Name}}.log
+`
```
Comment (AlekSi) [inline service_supervisord_linux.go:170]: @delgod The most important part for you to check ^

**Review by delgod — APPROVED** [review]
LGTM

File: service_supervisord_linux.go [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"text/template"
+	"time"
+)
+
+func isSupervisord() bool {
+	if _, err := os.Stat("/usr/bin/supervisord"); err == nil {
```
Comment (arvenil) [inline service_supervisord_linux.go:14]: I think adding ENV variable e.g. `ENABLE_SUPERVISORD` would be good idea. Then push `linux-supervisord` to the top of the list. This way you could use it for pmm-managed and also we can play with it with pmm-client as undocumented experiment.

File: service_supervisord_linux.go (hunk 1 above)
Comment (delgod) [reply service_supervisord_linux.go:170]: looks good, sorry for delay!

**Review by delgod — APPROVED** [review]
👍 thank you

**Comment by AlekSi** [comment]
@arvenil That one is wasted? Please close if yes.

**Pull request closed without merging**